Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage

```diff
@@ -1522,7 +1522,7 @@
 									<key>name</key>
 									<string>entity.name.function.matlab</string>
 									<key>match</key>
-									<string>[a-zA-Z][a-zA-Z0-9_.]*(?=[^a-zA-Z0-9_.])</string>
+									<string>[a-zA-Z][a-zA-Z0-9_]*(?&gt;\.[a-zA-Z0-9_]+)*</string>
```

Comment:
The previous regex will result in `functionName...` to be fully tokenized as the function name. 
The fix correctly parses line continuations, but still allows for correct tokenization of setter and getter calls e.g. `get.propertyName`.